core-api/src/test/java/com/optimizely/ab/optimizelyjson/types/MD1.java

```diff
@@ -0,0 +1,23 @@
+/**
+ *
+ *    Copyright 2020, Optimizely and contributors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package com.optimizely.ab.optimizelyjson.types;
+
+public class MD1 {
```

Comment:
It's not clear what these test classes are supposed to represent. It might be clearer if you subclass these under something like `TestTypes.MD1` etc and add some doc strings.